src/ARMJIT_Memory.cpp

```diff
@@ -701,7 +712,13 @@ void Init()
     FastMem7Start = MemoryBase + AddrSpaceSize;
     MemoryBase = MemoryBase + AddrSpaceSize*2;
 
-    MemoryFile = memfd_create("melondsfastmem", 0);
+    #ifdef __APPLE__
+        char* fastmem_pid_name = new char[snprintf(NULL, 0, "melondsfastmem%d", getpid()) + 1];
```

Comment:
just noticed, please make this camelCase instead of snake_case

Reply:
right, will do.